lib/fog/vsphere/requests/compute/list_networks.rb

```diff
@@ -20,10 +20,11 @@ def raw_networks(datacenter_name)
 
         def network_attributes network, datacenter
           {
-            :id         => managed_obj_id(network),
-            :name       => network.name,
-            :accessible => network.summary.accessible,
-            :datacenter => datacenter,
+            :id            => managed_obj_id(network),
+            :name          => network.name,
+            :accessible    => network.summary.accessible,
+            :datacenter    => datacenter,
+            :virtualswitch => network.class.to_s == "DistributedVirtualPortgroup" ? network.config.distributedVirtualSwitch.name : nil
```

Comment:
We can fix this after, but calling `class.to_s` seems like an odd pattern to me. `class.name` is the typical way I've seen things done like this.